src/metabase/query_processor/pivot/impl/common.clj

```diff
@@ -0,0 +1,149 @@
+(ns metabase.query-processor.pivot.impl.common
```

Comment:
I moved the stuff used by both the old pivot QP implementation and the new implementation from `metabase.query-processor.pivot` to `metabase.query-processor.pivot.impl.common`